docs/CHANGELOG.md

```
@@ -0,0 +1,38 @@
+- Updated dependencies to the latest versions.
+
+### Fixed
+- Fixed broken navigation on mobile.
+
+## [1.1.0] - 2023-12-15
+### Added
+- Introduced a search bar feature.
+
+<!--  -->
+
+### Added 
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)